apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/domain-step/index.tsx

```diff
@@ -14,20 +14,20 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
  */
 import LaunchStepContainer, { Props as LaunchStepProps } from '../../launch-step';
 import { LAUNCH_STORE } from '../../stores';
-import { useSite, useDomainSearch } from '@automattic/launch';
+import { useDomainSelection, useSiteDomains, useDomainSearch } from '@automattic/launch';
 
 import { FLOW_ID } from '../../constants';
 import './styles.scss';
 
 const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onNextStep } ) => {
-	const { plan, domain } = useSelect( ( select ) => select( LAUNCH_STORE ).getState() );
-	const { currentDomainName } = useSite();
+	const { plan } = useSelect( ( select ) => select( LAUNCH_STORE ).getState() );
+	const { currentDomain } = useDomainSelection();
+	const { siteSubdomain } = useSiteDomains();
 	const { domainSearch, setDomainSearch } = useDomainSearch();
 
 	const { setDomain, unsetDomain, unsetPlan, confirmDomainSelection } = useDispatch( LAUNCH_STORE );
 
 	const handleNext = () => {
-		confirmDomainSelection();
```

Comment:
Hey @razvanpapadopol , would you mind giving some context around these changes? They seem to affect Step by Step flow, but in the PR description I couldn't find any mention of them, and so I don't know what to test for

Reply:
It's the same change for both flows: 
> Don't preselect subdomain anymore. An explicit selection is now needed in order for Plans step to be highlighted 

and it was reviewed and merged in https://github.com/Automattic/wp-calypso/pull/47912

In Step by Step flow, before this change, the user was able to confirm the domain selection by clicking Next button. Now since the subdomain is not selected anymore, they need to select it.

Reply:
Got it, thank you!